bin/commands/migrate.js

```diff
@@ -13,7 +13,7 @@ const { migratePlugin, migrateApiFolder, migrateDependencies, migrateApplication
 // Global utils
 const { isPathStrapiApp, logger, isCleanGitRepo, promptUser } = require('../../lib/global/utils');
 
-const migrate = async (type, path, pathForV4Plugin) => {
+const migrate = async (type, path, pathForV4Plugin, cliOptions) => {
```

Comment:
I think we should change this to accept an object `{type, path, pathForV4Plugin, cliOptions}`